src/main/java/org/openstreetmap/atlas/geography/atlas/geojson/LineDelimitedGeoJsonConverter.java

```diff
@@ -0,0 +1,221 @@
+            Optionality.OPTIONAL, String.valueOf(DEFAULT_THREADS));
+
+    /**
+     * We only want positive (master) edges, because the negative edge can be derived at the
+     * application level, and this encodes extraneous data that can be easily derived by the map
+     * viewer. For relations, we only want multipolygon relations, as the rest can be derived from
+     * their members.
+     */
+    private static final Predicate<AtlasEntity> ENTITY_PREDICATE = entity ->
+    {
+        // We only want positive atlas entities. No negative ids.
+        if (ItemType.EDGE.equals(entity.getType()))
+        {
+            final Edge edge = (Edge) entity;
+            if (!edge.isMasterEdge())
+            {
+                return false;
+            }
+        }
+
+        // We only want multipolygon relations
+        if (ItemType.RELATION.equals(entity.getType()))
+        {
+            return Validators.isOfType(entity, RelationTypeTag.class, RelationTypeTag.MULTIPOLYGON);
+        }
+
+        // Because we're writing the multipolygon relations, we don't want to also write the area
+        // components that are pieces of the multipolygon relation.
+        if (ItemType.AREA.equals(entity.getType()))
+        {
+            final Set<Relation> relations = entity.relations();
+            if (relations.size() > 0)
+            {
+                return relations.stream().noneMatch(relation -> Validators.isOfType(relation,
+                        RelationTypeTag.class, RelationTypeTag.MULTIPOLYGON));
+            }
+        }
+
+        return true;
+    };
+
+    /**
+     * If we are rendering vector tiles, we may want to examine various tags of a given atlas entity
+     * and make decisions for the layer name, min zoom, and max zoom for the feature. Depending on
+     * your vector tile renderer, as well as map data visualization needs, you can override this
+     * BiConsumer to mutate your JSON object as you see fit.
+     */
+    private BiConsumer<AtlasEntity, JsonObject> jsonMutator = (atlasEntity, feature) ->
+    {
+    };
+
+    protected void setJsonMutator(final BiConsumer<AtlasEntity, JsonObject> jsonMutator)
+    {
+        this.jsonMutator = jsonMutator;
+    }
+
+    public static void main(final String[] args)
+    {
+        new LineDelimitedGeoJsonConverter().run(args);
+    }
+
+    @Override
+    protected int onRun(final CommandMap command)
+    {
+        final Time time = Time.now();
+        final Path atlasDirectory = (Path) command.get(ATLAS_DIRECTORY);
+        final Path geojsonDirectory = (Path) command.get(GEOJSON_DIRECTORY);
+        final Boolean overwrite = (Boolean) command.get(OVERWRITE);
+        final int threads = (Integer) command.get(THREADS);
+
+        if (overwrite)
+        {
+            try
+            {
+                FileUtils.deleteDirectory(geojsonDirectory.toFile());
+            }
+            catch (final IOException noDelete)
+            {
+                logger.warn(
```

Comment:
Nit: please add `noDelete` as the last parameter in the logger, this will ensure the stack trace is available in the logs.

Reply:
done 👍